include/clang/Interpreter/InterOp.h

```diff
@@ -1,5 +1,5 @@
 //--------------------------------------------------------------------*- C++ -*-
-// CLING - the C++ LLVM-based InterpreterG :)
+// CLING - the C++ LLVM-based Interpreter :)
```

Comment:
We should change the file preamble separately. I can do that later outside of this PR.

Reply:
change reverted